account_payment_widget_amount/models/account_move_line.py

```diff
@@ -1,5 +1,3 @@
-# Copyright 2018-2021 ForgeFlow S.L.
```

Comment:
You should not remove copyright section

Reply:
Ok, i am working on some new test cases.